redisinsight/api/src/modules/database/repositories/local.database.repository.ts

```diff
@@ -205,4 +218,20 @@ export class LocalDatabaseRepository extends DatabaseRepository {
 
     return decryptedEntity;
   }
+
+  /**
+   * Find existing entity by array of fields
+   * @param database
+   * @param fields
+   * @private
+   */
+  private async findEntity(database: Database, fields: string[]): Promise<Database | null> {
```

Comment:
I don't see that you added this method into common databse.repository interface when it should be there too

Reply:
it is a private method, like near methods `decryptEntity` and `encryptEntity`